jaxrs-server-security/build.gradle

```diff
@@ -1,6 +1,12 @@
 plugins {
     id "io.micronaut.build.internal.module"
 }
+
+repositories {
```

Comment:
Use a base module to reduce duplication. see https://github.com/micronaut-projects/micronaut-aws/pull/1445